TPCCAGPUTracking/CMakeLists.txt

```diff
@@ -196,6 +196,10 @@ if(ALITPCCOMMON_BUILD_TYPE STREQUAL "ALIROOT")
     # Additional compilation flags
     set_target_properties(${MODULE} PROPERTIES COMPILE_FLAGS "")
 
+    # OpenMP support
+    find_package(OpenMP)
+    target_link_libraries(${MODULE} OpenMP::OpenMP_CXX)
```

Comment:
we cannot change the AliRoot cmake here, you can have this change only locally, but not push it.
In order not to break AliRoot, please also add an
`#ifdef HLTCA_HAVE_OPENMP`
before all omp pragmas and omp_ API calls.